demo/spec/requests/preference_page_spec.rb

```diff
@@ -79,10 +79,10 @@
     end
 
     context "when localhost" do
```

Comment:
This is not really related to this PR - just something that slipped me in the previous one.